### PR TITLE
Buffs syndicate bundles.

### PR DESCRIPTION
**Pull request by DGamerL**
## What Does This PR Do
Buffs the following bundles:
Hacker:
- Gained: Camera bug
- Gained: Prototype freedom bio-chip
- Gained: Syndie AI lawboard
- Gained: Access tuner
- Lost: thermal goggles

Grenadier:
- Gained: Box of frag grenades
- Gained: Box of syndicate donks

Spy:
- Gained: Box of chameleon gear
- Gained: Garrote
- Gained: Access tuner
- Lost: Chameleon jumpsuit and mask

Agent 13:
- Gained: False bottom briefcase

The name's Bundle, Bond Bundle:
- Gained: Pen bomb - Reskinned syndieminibomb
- Lost: DNA scrambler

Payday:
- Gained: Prototype freedom bio-chip

Bio-chip bundle:
- Gained: Stealth bio-chip
- Lost: Uplink bio-chip

Metroid bundle:
- Gained: Adrenaline bio-chip
- Lost: Box of syndicate donks

Additionally, all stechkins have been upgraded to be their boxed variant with two magazines.

## Why It's Good For The Game
With the addition of #23465, two things have come to light:
- The implant bundle is one of the bundles of all time
- Bundles need to be buffed/changed
The only bundle I have slight doubts about is the Agent 13 one, but that is what feedback is for.

## Testing
Compiled, bundles were working as intended.
The pen bomb doesn't lose it's sprite when activated.

## Changelog
:cl:
tweak: Tweaked various traitor bundles, with the majority being buffed.
/:cl:

### CODE REVIEW

**Comment by Danchi299** [comment]
can't wait for Security Officer to check somebody's Pen for E-Dagger and cause Sabotage

**Comment by Danchi299** [comment]
while you are here, you might want to move `/obj/item/multitool/ai_detect` into `/obj/item/storage/belt/military/traitor/hacker,`

File: code/game/objects/items/weapons/grenades/syndieminibomb.dm [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /obj/item/grenade/syndieminibomb
-	desc = "A syndicate manufactured explosive used to sow destruction and chaos"
 	name = "syndicate minibomb"
+	desc = "A syndicate manufactured explosive used to sow destruction and chaos"
```
Comment (Henri215) [inline code/game/objects/items/weapons/grenades/syndieminibomb.dm:3]:
```suggestion
	desc = "A syndicate manufactured explosive used to sow destruction and chaos."
```

**Comment by DGamerL** [comment]
> while you are here, you might want to move `/obj/item/multitool/ai_detect` into `/obj/item/storage/belt/military/traitor/hacker,`

Good call, makes it easier. Done!

File: code/game/objects/items/weapons/grenades/syndieminibomb.dm [hunk 2]
```diff
@@ -24,3 +24,10 @@
 		to_chat(user, "<span class='userdanger'>As you activate the bomb, it emits a substance that sticks to your hand! It won't come off!</span>")
 		to_chat(user, "<span class='sans'>Uh oh.</span>")
 	. = ..()
+
+/obj/item/grenade/syndieminibomb/pen
+	name = "pen"
+	desc = "It's a normal black ink pen."
+	icon = 'icons/obj/bureaucracy.dmi'
+	icon_state = "pen"
+	item_state = "pen"
```
Comment (Sirryan2002) [inline code/game/objects/items/weapons/grenades/syndieminibomb.dm:33]: I recommend adding more functionality to this. This is extremely inconspicuous, so someone will be confused when they just randomly blow up. Perhaps adding a longer delay, beeping noises, and a custom priming message will go a long way to improving it.

Comment (Qwertytoforty) [reply code/game/objects/items/weapons/grenades/syndieminibomb.dm:33]: That or do what the trick minibomb does, put it in a clearly labled box, so the original traitor knows what it is, but can be used to trick others, and not be really clear when they throw it at someone with the beeping

Comment (DGamerL) [reply code/game/objects/items/weapons/grenades/syndieminibomb.dm:33]: I'll take those in mind, I'll make something good out of this

**Comment by Dafrek** [comment]
Nice looking changes imo, although grenadier bundle with meth might be a little much and the implant bundle now automatically coming with a stealth implant seems like its going to make it the most painful bundle to play against.

**Comment by DGamerL** [comment]
> Nice looking changes imo, although grenadier bundle with meth might be a little much and the implant bundle now automatically coming with a stealth implant seems like its going to make it the most painful bundle to play against.

The grenadier bundle needed something, it is a very weak bundle as it currently stands.

On the implant bundle, don't forget it came with an uplink bio-chip which came with 50 TC pre-installed. You could already buy the stealth bio-chip and have 5 TC spare.

**Comment by DGamerL** [comment]
>  Adding survivability/escapability without removing some of the firepower just does not make that much sense IMO.

Meth donks are, as it currently stands, a very hard tradeoff. They help, but can easily kill you via either brain damage or hunger.

> Of course, you could already buy the stealth implant with the implant bundle previously, but this change forces people to receive a painful implant to play against if they choose that bundle instead of choosing what to buy. If you want to nerf the implant bundle then giving it one of the best implants for stealth might not be the way to go.

It's still a nerf, and the bio-chip bundle is usually only bio-chips. Don't forget that this is it for your TC. You still need to get something to pair with this bundle.
What's also factored in is the potential for discounts, and with the 50 TC, you can easily afford those extra cheap items.

**Pull request merged**